src/views/GovernanceTaskList/index.tsx

```diff
@@ -97,11 +98,17 @@ function GovernanceTaskList() {
                     {t('button.saveAndExit')}
                   </UswdsReactLink>
                 </div>
-                <div className="margin-top-1">
-                  <Button type="button" unstyled className="text-error">
-                    {t('button.removeYourRequest')}
-                  </Button>
-                </div>
+                {/* Only allow removal if the intake form has not yet been submitted */}
+                {[
+                  ITGovIntakeFormStatus.READY,
+                  ITGovIntakeFormStatus.IN_PROGRESS
+                ].includes(systemIntake.itGovTaskStatuses.intakeFormStatus) && (
```

Comment:
The AC of EASI-3139 call out that this fix should work for v1 and v2 requests. My understanding is that `itGovTaskStatuses` will only be set on v2 requests.

I _think_ that Steven's PR here will actually change that, so if you don't want to have to write v1 AND v2 logic to handle this, maybe we just try and get this PR merged first - https://github.com/CMSgov/easi-app/pull/2148

@StevenWadeOddball - thoughts on this?